### PR TITLE
[WIP][Checkout] Paypal payment

### PR DESCRIPTION
**Pull request by Arminek**

### CODE REVIEW

File: src/Sylius/Behat/Context/Setup/PaymentContext.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
 namespace Sylius\Behat\Context\Setup;
 
 use Behat\Behat\Context\Context;
+use Proxies\__CG__\Sylius\Component\Payment\Model\PaymentMethod;
```
Comment (michalmarcinkowski) [inline src/Sylius/Behat/Context/Setup/PaymentContext.php]: Should be removed

Comment (pamil) [inline src/Sylius/Behat/Context/Setup/PaymentContext.php]: ?

Comment (stloyd) [inline src/Sylius/Behat/Context/Setup/PaymentContext.php]: It's not proper class usage, it's proxy of the model not model itself.

File: app/config/config_test.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 imports:
     - { resource: config.yml }
+    - { resource: secret_parameters.php }
```
Comment (stloyd) [inline app/config/config_test.yml]: I would prefer to add configuration with such parameters not the "hacky hack" that uses envs

File: app/config/secret_parameters.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+$paypalAccountName = getenv('SECRET_BEHAT_PAYPAL_ACCOUNT_NAME');
+$paypalAccountPassword = getenv('SECRET_BEHAT_PAYPAL_ACCOUNT_PASSWORD');
+
+if (!$paypalAccountName || !$paypalAccountName) {
+    $container->setParameter('sylius.behat.paypal_account_name', null);
+    $container->setParameter('sylius.behat.paypal_account_password', null);
+}
+
+$container->setParameter('sylius.behat.paypal_account_name', $paypalAccountName);
```
Comment (stloyd) [inline app/config/secret_parameters.php]: Why setting this twice?

File: features/shop/checkout_with_paypal_payment.feature [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    Background:
+        Given that store is operating on the France channel
+          And default currency is "EUR"
+          And there is user "john@example.com" identified by "password123"
+          And catalog has a product "PHP T-Shirt" priced at $19.99
+          And store allows paying "PayPal Express Checkout"
+          And store has free shipping method
+          And I am logged in as "john@example.com"
+
+@javascript
```
Comment (stloyd) [inline features/shop/checkout_with_paypal_payment.feature]: This tag could be set at top as it contains only js based tests.

**Comment by okwinza** [comment]
Why close this @Arminek  ?

**Comment by pamil** [comment]
@okwinza there's brand new PR #4054 with the same feature :)

**Pull request closed without merging**